data/test/scenarios/manual_tests/test_max_menu_items.cfg

```diff
@@ -1,5 +1,36 @@
 #textdomain wesnoth-test
-{GENERIC_UNIT_TEST "test_max_menu_items" (
+
+[test]
+    name = _ "Unit Test test_max_menu_items"
+    map_file=test/maps/generic_unit_test.map
+    turns = -1
+    id = test_max_menu_items
+    
```

Comment:
Trailing whitespace on a couple of lines in this file.

I think it could go back to using `GENERIC_UNIT_TEST`, if the change was just to make it use `[multiplayer]`.

Reply:
Updated.